README.md

```diff
@@ -53,7 +54,26 @@ sudo apt install python3-dev libpq-dev
 
 8. Now run `python manage.py runserver`. Go to http://127.0.0.1:8000/ in your browser. You should be able to access our project.
 
+## Heroku Deployment
+1. Install Heroku CLI
+   1. Instructions can be found here: https://devcenter.heroku.com/articles/heroku-cli
+       * For macOS: `brew tap heroku/brew && brew install heroku`
+       * For Ubuntu/Debian `curl https://cli-assets.heroku.com/install-ubuntu.sh | sh`
+       * For Arch Linux `yay -S heroku-cli`
+       * For Windows: Use this link address to install 64bit installer https://cli-assets.heroku.com/heroku-x64.exe OR Use this link address to install 32 bit installer https://cli-assets.heroku.com/heroku-x86.exe
+   2.  Verify your installation: Running `heroku --version`, output should be like `heroku/x.y.z`
+
+2. Heroku Deployment
+   1. Create an account in [Heroku.](https://signup.heroku.com/) 
+   2. Login your Heroku account in terminal by using `heroku login`
+   3. Go to http://dashboard.heroku.com/new-app to create a new Heroku instance
+      * In the instance page, go to "Resources", in the add-ons search postgres, choose Heroku-Postgres, then click "Provision" to add it.
```

Comment:
I think it would be better to move the creation of Postgres db addon to a single step.

Reply:
I think this part would be better to stay here, because when I creating everything from scratch, this step is the natural step after creating a Heroku instance. So that user don't need to open heroku admin page to add that add-on again in the step later.

Reply:
I mean something like this: Step 3 is for creating an instance; Step 4 is for creating and linking a Postgres database; Step 5 will resume to regular steps onward.

Reply:
I get your idea, changes have been made

Reply:
I get your idea, changes have been made

Reply:
Saw it. Awesome.